collectors/0/pxc-collector.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+"""
+TCollector for Percona XtraDB Clusters
+
+It's possible to query every single 'wsrep'-Variable.
+If you want to add more variables simply edit the pxcconf.py
+
+ATTENTION: Only tested on Debian/Ubuntu systems.
+"""
+
+import MySQLdb as mysql
+import time
+import sys
+import os.path
+from collectors.etc import pxcconf
+from collectors.lib import utils
+
+__author__     = "Kai Laufer"
+__license__    = "GPL"
+__version__    = "1.0.0"
+__email__      = "mail@kai-laufer.de"
+
+""" You can find these functions in etc/pxcconf.py """
+prefix      = pxcconf.getPrefix()
+interval    = pxcconf.getInterval()
+galeraFile  = pxcconf.getGaleraFile()
+login       = pxcconf.getUserPassword()
+myMap       = pxcconf.getKeyMap()
+mysqlUser   = login[0]
+mysqlPasswd = login[1]
+mysqlHost   = login[2]
+
+def getRow():
+	db      = mysql.connect(host=mysqlHost, user=mysqlUser, passwd=mysqlPasswd)
```

Comment:
It would be nice to wrap this in a try and if there is a permanent issue with connecting, such as a bad username/password or hostname, you can exit 13 and tell tCollector not to run this one anymore.